installation failed with duplicate symbol

ОС Ubuntu 21.04

```sh
...
[100%] Linking C shared library tntkafka.so
/usr/bin/ld: CMakeFiles/tntkafka.dir/callbacks.c.o:/tmp/luarocks_kafka-scm-1-e4c9Kb/kafka/kafka/common.h:22: повторное определение «producer_label»; CMakeFiles/tntkafka.dir/tnt_kafka.c.o:/tmp/luarocks_kafka-scm-1-e4c9Kb/kafka/kafka/common.h:22: здесь первое определение
/usr/bin/ld: CMakeFiles/tntkafka.dir/callbacks.c.o:/tmp/luarocks_kafka-scm-1-e4c9Kb/kafka/kafka/common.h:21: повторное определение «consumer_msg_label»; CMakeFiles/tntkafka.dir/tnt_kafka.c.o:/tmp/luarocks_kafka-scm-1-e4c9Kb/kafka/kafka/common.h:21: здесь первое определение
/usr/bin/ld: CMakeFiles/tntkafka.dir/callbacks.c.o:/tmp/luarocks_kafka-scm-1-e4c9Kb/kafka/kafka/common.h:20: повторное определение «consumer_label»; CMakeFiles/tntkafka.dir/tnt_kafka.c.o:/tmp/luarocks_kafka-scm-1-e4c9Kb/kafka/kafka/common.h:20: здесь первое определение
/usr/bin/ld: CMakeFiles/tntkafka.dir/consumer.c.o:/tmp/luarocks_kafka-scm-1-e4c9Kb/kafka/kafka/common.h:20: повторное определение «consumer_label»; CMakeFiles/tntkafka.dir/tnt_kafka.c.o:/tmp/luarocks_kafka-scm-1-e4c9Kb/kafka/kafka/common.h:20: здесь первое определение
/usr/bin/ld: CMakeFiles/tntkafka.dir/consumer.c.o:/tmp/luarocks_kafka-scm-1-e4c9Kb/kafka/kafka/common.h:21: повторное определение «consumer_msg_label»; CMakeFiles/tntkafka.dir/tnt_kafka.c.o:/tmp/luarocks_kafka-scm-1-e4c9Kb/kafka/kafka/common.h:21: здесь первое определение
/usr/bin/ld: CMakeFiles/tntkafka.dir/consumer.c.o:/tmp/luarocks_kafka-scm-1-e4c9Kb/kafka/kafka/common.h:22: повторное определение «producer_label»; CMakeFiles/tntkafka.dir/tnt_kafka.c.o:/tmp/luarocks_kafka-scm-1-e4c9Kb/kafka/kafka/common.h:22: здесь первое определение
/usr/bin/ld: CMakeFiles/tntkafka.dir/consumer_msg.c.o:/tmp/luarocks_kafka-scm-1-e4c9Kb/kafka/kafka/common.h:21: повторное определение «consumer_msg_label»; CMakeFiles/tntkafka.dir/tnt_kafka.c.o:/tmp/luarocks_kafka-scm-1-e4c9Kb/kafka/kafka/common.h:21: здесь первое определение
/usr/bin/ld: CMakeFiles/tntkafka.dir/consumer_msg.c.o:/tmp/luarocks_kafka-scm-1-e4c9Kb/kafka/kafka/common.h:22: повторное определение «producer_label»; CMakeFiles/tntkafka.dir/tnt_kafka.c.o:/tmp/luarocks_kafka-scm-1-e4c9Kb/kafka/kafka/common.h:22: здесь первое определение
/usr/bin/ld: CMakeFiles/tntkafka.dir/consumer_msg.c.o:/tmp/luarocks_kafka-scm-1-e4c9Kb/kafka/kafka/common.h:20: повторное определение «consumer_label»; CMakeFiles/tntkafka.dir/tnt_kafka.c.o:/tmp/luarocks_kafka-scm-1-e4c9Kb/kafka/kafka/common.h:20: здесь первое определение
/usr/bin/ld: CMakeFiles/tntkafka.dir/producer.c.o:/tmp/luarocks_kafka-scm-1-e4c9Kb/kafka/kafka/common.h:22: повторное определение «producer_label»; CMakeFiles/tntkafka.dir/tnt_kafka.c.o:/tmp/luarocks_kafka-scm-1-e4c9Kb/kafka/kafka/common.h:22: здесь первое определение
/usr/bin/ld: CMakeFiles/tntkafka.dir/producer.c.o:/tmp/luarocks_kafka-scm-1-e4c9Kb/kafka/kafka/common.h:21: повторное определение «consumer_msg_label»; CMakeFiles/tntkafka.dir/tnt_kafka.c.o:/tmp/luarocks_kafka-scm-1-e4c9Kb/kafka/kafka/common.h:21: здесь первое определение
/usr/bin/ld: CMakeFiles/tntkafka.dir/producer.c.o:/tmp/luarocks_kafka-scm-1-e4c9Kb/kafka/kafka/common.h:20: повторное определение «consumer_label»; CMakeFiles/tntkafka.dir/tnt_kafka.c.o:/tmp/luarocks_kafka-scm-1-e4c9Kb/kafka/kafka/common.h:20: здесь первое определение
/usr/bin/ld: CMakeFiles/tntkafka.dir/common.c.o:/tmp/luarocks_kafka-scm-1-e4c9Kb/kafka/kafka/common.h:22: повторное определение «producer_label»; CMakeFiles/tntkafka.dir/tnt_kafka.c.o:/tmp/luarocks_kafka-scm-1-e4c9Kb/kafka/kafka/common.h:22: здесь первое определение
/usr/bin/ld: CMakeFiles/tntkafka.dir/common.c.o:/tmp/luarocks_kafka-scm-1-e4c9Kb/kafka/kafka/common.h:21: повторное определение «consumer_msg_label»; CMakeFiles/tntkafka.dir/tnt_kafka.c.o:/tmp/luarocks_kafka-scm-1-e4c9Kb/kafka/kafka/common.h:21: здесь первое определение
/usr/bin/ld: CMakeFiles/tntkafka.dir/common.c.o:/tmp/luarocks_kafka-scm-1-e4c9Kb/kafka/kafka/common.h:20: повторное определение «consumer_label»; CMakeFiles/tntkafka.dir/tnt_kafka.c.o:/tmp/luarocks_kafka-scm-1-e4c9Kb/kafka/kafka/common.h:20: здесь первое определение
collect2: error: ld returned 1 exit status
gmake[2]: *** [kafka/CMakeFiles/tntkafka.dir/build.make:196: kafka/tntkafka.so] Ошибка 1
gmake[1]: *** [CMakeFiles/Makefile2:113: kafka/CMakeFiles/tntkafka.dir/all] Ошибка 2
gmake: *** [Makefile:149: all] Ошибка 2

Error: Failed installing dependency: http://rocks.tarantool.org/kafka-scm-1.rockspec - Build error: Failed building.
   ⨯ Failed to install rocks: Failed to run 
/usr/bin/tarantoolctl rocks make
```

--- a/kafka/common.h
+++ b/kafka/common.h
@@ -17,9 +17,9 @@
 
 #include <tarantool/module.h>
 
-const char* const consumer_label;
-const char* const consumer_msg_label;
-const char* const producer_label;
+extern const char* const consumer_label;
+extern const char* const consumer_msg_label;
+extern const char* const producer_label;
 
 int lua_librdkafka_version(struct lua_State *L);
 

